src/fparser/two/tests/fortran2003/test_format_item_list.py

```diff
@@ -134,6 +134,56 @@ def test_hollerith_only_spaces(f2003_create, monkeypatch):
     )
 
 
+def test_hollerith_omitted_comma_before(f2003_create, monkeypatch):
+    """Check that a hollerith item is parsed correctly when preceeded by a
+    slash or colon without a separating comma.
+
+    """
+    from fparser.two import utils
```

Comment:
Please could you move this import up to the top and replace the use of `format` with f-strings in the new tests (as suggested by pylint).